### PR TITLE
Add Trove Classifier Modernizer

### PR DESCRIPTION
**Pull request by aht007**
JIRA ISSUE: [BOM-3278](https://openedx.atlassian.net/browse/BOM-3278)

## DESC
This PR adds a new modernizer to update the Trove Classifiers in the setup.py file. The modernizer removes outdated classifiers and adds new ones if needed.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -36,6 +36,11 @@ def get_version(*file_paths):
         'Environment :: Console',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Apache Software License'
+        'Framework :: Django',
```
Comment (aht007) [inline setup.py:39]: These are just testing changes and will be removed once PR is finalized

File: edx_repo_tools/codemods/django3/setup_file_modernizer.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    def _write_data_to_file(self, file_data) -> None:
+        with open(self.setup_file_path, 'w') as setup_file:
+            setup_file.write(file_data)
+
+    def update_setup_file(self) -> None:
+        self._apply_regex_operations()
+
+
+@click.command()
```
Comment (Jawayria) [inline edx_repo_tools/codemods/django3/setup_file_modernizer.py:52]: Adding this decorator will just add a help message and nothing else. You should add a new entry point in setup.py file to run this modernizer via command. Moreover we usually prefer to accept path from command line in addition to setting a default path. This provides more flexibility while running commands. For reference, you can look into other django3 codemods we already have. 
[command line option in tox modernizer](https://github.com/openedx/repo-tools/blob/8762e5a6a9a6a48be83272bb14248798c6a9231f/edx_repo_tools/codemods/django3/tox_modernizer.py#L110)

Comment (aht007) [reply edx_repo_tools/codemods/django3/setup_file_modernizer.py:52]: Yeah The entry point changes were pending and I have added that. Also I have made necessary changes related to accepting path from command line. You can review again.

**Review by Jawayria — APPROVED** [review]
Bump the version and create a release when PR gets merged. The PR looks good to me.

**Review by mraarif — APPROVED** [review]

**Pull request merged**